### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by goruha**

### CODE REVIEW

File: makes/Makefile.copyright [hunk 1]
```diff
@@ -0,0 +1,22 @@
+COPYRIGHT_CMD ?= docker run --rm --volume `pwd`:$(COPYRIGHT_OUTPUT_DIR) osterman/copyright-header:latest
+COPYRIGHT_LICENSE ?= ASL2
+COPYRIGHT_HOLDER ?= Cloud Posse, LLC <hello@cloudposse.com>
+COPYRIGHT_YEAR ?= 2016
+COPYRIGHT_SOFTWARE ?= Github PAM
+COPYRIGHT_SOFTWARE_DESCRIPTION ?= Use GitHub authentication as PAM
```
Comment (osterman) [inline makes/Makefile.copyright]: replace it with "Example Software Description"

File: makes/Makefile.copyright [hunk 2]
```diff
@@ -0,0 +1,22 @@
+COPYRIGHT_CMD ?= docker run --rm --volume `pwd`:$(COPYRIGHT_OUTPUT_DIR) osterman/copyright-header:latest
+COPYRIGHT_LICENSE ?= ASL2
+COPYRIGHT_HOLDER ?= Cloud Posse, LLC <hello@cloudposse.com>
+COPYRIGHT_YEAR ?= 2016
```
Comment (osterman) [inline makes/Makefile.copyright]: replace with `$(shell date +%Y)

File: makes/Makefile.copyright [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+COPYRIGHT_SOFTWARE ?= Github PAM
```
Comment (osterman) [inline makes/Makefile.copyright]: replace with `Example App`

File: makes/Makefile.copyright (hunk 1 above)
Comment (goruha) [inline makes/Makefile.copyright]: Done

File: makes/Makefile.copyright (hunk 2 above)
Comment (goruha) [inline makes/Makefile.copyright]: Done

File: makes/Makefile.copyright (hunk 3 above)
Comment (goruha) [inline makes/Makefile.copyright]: Done

**Pull request closed without merging**